draft/draft-ietf-rtgwg-qos-model.xml

```diff
@@ -121,15 +121,15 @@
 
   <middle>
     <section anchor="intro" title="Introduction">
-      <t> This document defines a base YANG <xref target="RFC6020"/>
+      <t> This document defines a YANG <xref target="RFC6020"/>
       <xref target="RFC7950"/> model for Quality of Service (QoS)
       configuration and statistics parameters. Traffic Policy module 
       defines the basic building blocks to define a classifier, policy
       and target. QoS Action module defines QoS action related
       parameters. Traffic policy is augmented to include packet match
       and action parameters to define the Diffrentiated Services
       (DiffServ) policy, Queues policy and  Scheduler policy. Each
-      of these policies are defined as separate modules. The Diffserv
+      of these policies is defined as separate module. The Diffserv
```

Comment:
s/module/modules/

or undo the change, as you are still referring to multiple modules.